contrib/facter.ps1

```diff
@@ -127,6 +127,7 @@ if ($buildSource) {
     "--with-regex",
     "--with-log",
     "--with-locale",
+    "--with-chrono",
```

Comment:
Is this something we're going to want to pull into the linux/solaris builds as well, or is it windows specific?